modules/ROOT/pages/to-configure-api-settings.adoc

```diff
@@ -4,28 +4,24 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
-Anypoint Exchange lets you configure whether REST, SOAP, and HTTP API services are public or private in a Private Exchange or in an Exchange Portal. Typical services that you would set visibility for are proxies or policies associated with your API that you set using API Manager.
+Use Anypoint Exchange to configure whether REST, SOAP, and HTTP API services are public or private in a private Exchange or in an Exchange portal. For example, configure public or private visibility for proxies or policies associated with your API and set using API Manager.
 
-After you register an API from Exchange in API Manager, the instances for the API appear in Exchange in the API Instances list in the left navigation bar.
+After you register an API from Exchange in API Manager, the instances for the API appear in Exchange in the API instances list in the navigation bar.
 
-You can also create an API instance if your API is managed by another API management program outside of Anypoint Platform.
+Exchange also creates API instances for APIs managed by another API management program outside of Anypoint Platform.
 
-Access differs by Exchange type:
+Access differs by Exchange type. In a private Exchange, the API instances list appears for HTTP, REST, and SOAP APIs. In an Exchange portal, the API instances list appears for SOAP and HTTP APIs. All instances in an Exchange portal are read only.
 
-* In a Private Exchange, API Instances appears for HTTP, REST, and SOAP APIs. If you are an asset administrator, you can change the visibility of API instances and add new instances:
-+
-** If you want an instance public, change the Visibility indicator.
-** You can edit the instance by clicking the pencil icon, or delete the instance by clicking the X icon.
-+
-* In an Exchange Portal, API Instances appears for SOAP and HTTP APIs. All instances in an Exchange Portal are read only.
+Visibility for each instance is either private or public. All users in a business group's Exchange portal can see a public instance. Users can see a private instance if the asset is shared with them or if they have an Anypoint Platform permission that grants access.
```

Comment:
>All users in a business group's Exchange portal can see a public instance

The wording here seems off since Exchange Public Portals are not scoped to Bussiness Groups. We could change this to
"All users accessing an Exchange Public Portal can see a public API instance."

Reply:
Thanks! Fixed.